engine/crates/engine-v2/schema/src/builder/graph.rs

```diff
@@ -206,13 +205,7 @@ impl<'a> GraphBuilder<'a> {
             .map(|federated_enum| EnumDefinition {
                 name: federated_enum.name.into(),
                 description: None,
-                value_ids: {
-                    let range = idmap.get_range(federated_enum.values);
-                    self.graph.enum_value_definitions[Range::<usize>::from(range)]
```

Comment:
That breaks the references, so it had to go. I don't know why we sorted alphabetically, please comment if that breaks assumptions.

Reply:
Ok the test failures pointed that out to me. We now have to do a linear search instead of a binary search to find an enum value by name. That probably doesn't matter other than in very pathological cases. I'd keep it in because there are other ways to speed up this lookup if we care, but please do voice any disagreement.

Reply:
I think you can stay with 1). But I don't think 2) is that hard, it's having another `idmap`, but instead of keeping only skipped IDs, you have a dense array mapping each id to their new one. I wouldn't do 3).

Reply:
Hmm yes, you have a conversion step so there's a clear place where you can use that mapping. Good point.